src/charts/charts.service.spec.ts

```diff
@@ -49,12 +48,16 @@ describe('charts.service', () => {
 
   describe('getChartData', () => {
     const today = new Date();
+    today.setUTCHours(0, 0, 0, 0);
```

Comment:
If we want to be accurate working with days, it`s better to set start or end of it

Reply:
this one was flaky